src/pint/utils.py

```diff
@@ -486,8 +506,8 @@ def dmxparse(fitter, save=False):
         DMX_Errs.append(
             getattr(fitter.model, "DMX_{:04d}".format(ii)).uncertainty_value
         )
-        dmxr1 = getattr(fitter.model, "DMX_{:04d}".format(ii)).value
-        dmxr2 = getattr(fitter.model, "DMX_{:04d}".format(ii)).value
+        dmxr1 = getattr(fitter.model, "DMXR1_{:04d}".format(ii)).value
```

Comment:
@bshapiroalbert In this PR, I included this bug fix in dmxparse(). It was reading DMX_nnnn instead of DMXR1_nnnn and DMXR2_nnnn. I'm not sure how this ever worked? Did you verify that the results from this matched Paul's dmxparse?

Reply:
@paulray Thanks for fixing that, that's embarrassing that I didn't catch that. I did check this version against Paul's dmxparse and got consistent results. The R1 and R2 aren't used unless you want to plot something or print an output file, they're not used when actually doing the mean subtraction on the DMX values (DMX_nnnn) or getting the errors, since there's no error on R1/2. But that will give you incorrect MJDs for the DMX bins....

Reply:
Yeah, I noticed it when I tried to plot my DMX time series :-)